### PR TITLE
Implement get-alias

### PR DESCRIPTION
**Pull request by dolanske**
Added an endpoint which returns all available aliases from the database

### CODE REVIEW

File: backend/src/api/alias.rs [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+}
+
+#[derive(Serialize, Deserialize, Debug)]
+pub struct Alias {
+    name: String,
+    content: String,
+}
+
+async fn get_aliases<D: SqliteDatabase>(
+    Authorize(_username): Authorize,
```
Comment (Jokler) [inline backend/src/api/alias.rs:28]: This could be `Authorize(_): Authorize` instead but either is fine by me.

File: backend/src/api/alias.rs [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    Extension(state): Extension<Arc<AppState<D>>>,
+) -> Result<Json<Vec<Alias>>, Error> {
+    let conn = state.pool.get().await.context("Failed to get connection")?;
+
+    conn.interact(move |conn| {
+        let mut query = conn
+            .prepare("SELECT * FROM ALIASES")
```
Comment (Jokler) [inline backend/src/api/alias.rs:35]: I'm surprised to see that table names are case insensitive but I would prefer if only SQL commands were uppercase and table names were lowercase like in the database.

Comment (dolanske) [reply backend/src/api/alias.rs:35]: I'd argue that for this specific request I'd want to have all the available information for aliases.

Comment (Jokler) [reply backend/src/api/alias.rs]: That's not how it works though, you have to make code changes to the `Alias` struct anyway if you want to serve more data: https://github.com/Mavulp/hivefriends/blob/0aa883a0f4569f37c5410107e0c395f9d09fc3b1/backend/src/api/alias.rs#L22-L25

**Pull request merged**